service-python/pdfgenerator/src/lib/helper_functions.py

```diff
@@ -5,8 +5,18 @@
 from dateutil.relativedelta import relativedelta
 
 
-def pdf_helper_asthma(data):
-
+def set_empty_readings(value):
+    if value == 0:
```

Comment:
Rather than changing the central data model for LH and MAS data, we are setting incoming dashes from MAS to 0, and then setting it back to a dash before it goes onto the pdf. All integer values are turned need to be converted to  strings to allow for dashes.

Reply:
We are assuming no complete BP readings have value = 0.